src/reporter/reporter.py

```diff
@@ -104,7 +104,22 @@ def _validate_payload(payload):
             payload[attr].update({'value': None})
             log().warning(
                 'An entity update is missing value for attribute {}'.format(attr))
+  
 
+def _filter_empty_entities(payload):
+    log().info('Received payload: {}'.format(payload))
+    attrs = list(iter_entity_attrs(payload))
+    Flag = False
+    attrs.pop()
```

Comment:
Didn't we decide to get rid of `pop` here? It isn't needed if `_filter_empty_entities` gets called before we add the time index attribute to the entity, see my comment below :-)